lib/api/core/plugins/pluginsManager.js

```diff
@@ -431,9 +439,9 @@ function initControllers (controllers, routes, plugin, pluginName) {
           return false;
         }
 
-        if (['post', 'get'].indexOf(route.verb.toLowerCase()) === -1) {
+        if (['get', 'head', 'post', 'put', 'delete', 'connect', 'options', 'trace', 'patch'].indexOf(route.verb.toLowerCase()) === -1) {
```

Comment:
is the `httpRouter` already able to handle all these methods? As far as I can see, only `get`, `post`, `put` and `delete` are accepted.

Further, if I am not mistaken, the `options` verb has a specific behaviour within kuzzle &/|| the proxy.